theia-java-docker/Dockerfile

```diff
@@ -26,7 +26,8 @@ ARG version=latest
 ADD $version.package.json ./package.json
 ARG GITHUB_TOKEN
 RUN yarn --cache-folder ./ycache && rm -rf ./ycache
-RUN yarn theia build
+RUN yarn theia build && yarn cache clean
+RUN apt-get clean
```

Comment:
I do not think this does much good. IIRC for this to be worthwhile (i.e. save space), it would need to happen in the same ```RUN``` command as the one that created/updated the apt cache, else I think these files will still be present on that image layer, and just marked as removed in one of the following layers. 

So the "apt clean" command should probably be added to the "RUN apt-get update && ..." command on line 10.

Reply:
Should be fixed on the latest change.